src/plugins/elasticsearch/lib/health_check.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (plugin, server) {
     if (timeoutId) return;
 
     var myId = setTimeout(function () {
-      check().finally(function () {
+      Promise.try(check).finally(function () {
```

Comment:
`check` already returns a Promise, why'd you end up wrapping this in Promise.try?